cumulus/scripts/migrate_storage_to_genesis/identityMigration.js

```diff
@@ -59,6 +59,13 @@ const migrateIdentity = async (key, data, api) => {
 			}
 		});
 
+		let judgements = [];
+		decodedJson.judgements.forEach((judgement) => {
+			if (!('feePaid' in judgement[1])) {
+				judgements.push(judgement);
+			}
+		});
```

Comment:
```suggestion
		let judgements = [];
		console.log("before: ", decodedJson.judgements);
		decodedJson.judgements.forEach((judgement) => {
			if (!('feePaid' in judgement[1])) {
				judgements.push(judgement);
			}
		});
		console.log("after: ", judgements);
```

if you log it like ^, you get https://gist.github.com/muharem/41748f9ffb7cc8c58162d7fc0eb9c992

Reply:
Looks good, both for accounts that [_only_ have `FeePaid`](https://gist.github.com/muharem/41748f9ffb7cc8c58162d7fc0eb9c992#file-gistfile1-txt-L191-L192) and those that have [other judgements](https://gist.github.com/muharem/41748f9ffb7cc8c58162d7fc0eb9c992#file-gistfile1-txt-L207-L208).